### DISCUSSION TITLE
Fuzzy skin like color painting

### DISCUSSION THREAD

**Discussion by vincentbcd** [OrcaSlicer/OrcaSlicer#550] [Ideas]
Hello,

In a first time, thank you for your great job!
As the title says, it would be great to have the fuzzy skin in the same way that we paint our object with colors.

Thank you

**Comment by PLAmenPRINT** [comment]
I would like to have that option too.
I have open a Feature request ticket about that some time ago. (Fuzzy skin as Color painting. #269)
Is this difficult to be made?
I am ready to donate what is needed if that will speed up the process :)

**Reply by Ciantha** [reply]
Wanted to chime in that this would be a really neat feature. Wanted to fuzzy skin parts for my P1P enclosure but since there's no real "inside" I couldn't figure out a way to only get fuzzy skin on the outwards facing (once assembled) parts.

**Comment by Mowhok** [comment]
Jeah me too, i was just about to add another thread

**Comment by DARKFiB3R** [comment]
This would be great.

**Comment by fritsjan** [comment]
Yes please, fuzzy skin paint on would be a gamechanger. This way you can paint on different textures...

**Comment by XIII-IIIX** [comment]
I recently made the switch from Windows to Mint and have been having issues installing the .appimage. Luckily I found the nightly buils with a .flatpak!! Anyway, while I was doom-spiraling about not being able to install Orca, I downloaded Purse Sclicer and saw that they have a paint fuzzy skin option. Honestly, I would have stuck with it for that reason alone had better support for my printer. 

I do wonder what the lift would be to have this implemented in Orca.
